docs/contributor/telemetry-load-test/README.md

```diff
@@ -213,4 +213,75 @@ Each test scenario has its own test scripts responsible for preparing test scena
+
+```shell
+./run-load-test.sh -t fluentbit -n "2.2.1" -m true
+```
+
+4. To test the buffering and retry capabilities of 3 LogPipelines with simulated backend outages, run:
+
+```shell
+./run-load-test.sh -t fluentbit -n "2.2.1" -m true -b true
+```
+
+#### Test Results
+
+
+
+<div class="table-wrapper" markdown="block">
+
+| Version/Test |             Single Pipeline             |                                          |                                 |                      |               |             Multi Pipeline              |                                          |                                 |                      |               |      Single Pipeline Backpressure       |                                          |                                 |                      |               |       Multi Pipeline Backpressure       |                                          |                                 |                      |               |
+|-------------:|:---------------------------------------:|:----------------------------------------:|:-------------------------------:|:--------------------:|:-------------:|:---------------------------------------:|:----------------------------------------:|:-------------------------------:|:--------------------:|:-------------:|:---------------------------------------:|:----------------------------------------:|:-------------------------------:|:--------------------:|:-------------:|:---------------------------------------:|:----------------------------------------:|:-------------------------------:|:--------------------:|:-------------:|
+|              | Input Bytes Processing Rate/sec (KByte) | Output Bytes Processing Rate/sec (KByte) | Filesystem Buffer Usage (KByte) | Pod Memory Usage(MB) | Pod CPU Usage | Input Bytes Processing Rate/sec (KByte) | Output Bytes Processing Rate/sec (KByte) | Filesystem Buffer Usage (KByte) | Pod Memory Usage(MB) | Pod CPU Usage | Input Bytes Processing Rate/sec (KByte) | Output Bytes Processing Rate/sec (KByte) | Filesystem Buffer Usage (KByte) | Pod Memory Usage(MB) | Pod CPU Usage | Input Bytes Processing Rate/sec (KByte) | Output Bytes Processing Rate/sec (KByte) | Filesystem Buffer Usage (KByte) | Pod Memory Usage(MB) | Pod CPU Usage |
+|        2.2.1 |                  7294                   |                   3847                   |              80553              |       185,181        |    0.8,0.9    |                  2484                   |                   1205                   |              94435              |       321,325        |    0.8,0.8    |                  7611                   |                   665                    |              36704              |       208,219        |    0.8,0.8    |                  2501                   |                   231                    |              97754              |       325,320        |    0.8,0.8    |
```

Comment:
We see a output rate that is lower than the input rate and a significant file-system buffer usage already for the first test. It looks like this test case already creates a backpressure situation. Maybe try to reduce the generated amount of logs a bit.

Reply:
test result updated with new setup